packages/components/src/unit-control/index.js

```diff
@@ -158,6 +159,19 @@ function UnitControl(
 		/>
 	) : null;
 
+	let step = props.step;
+	// if no step prop has been passed
```

Comment:
I would replace these micro-level comments with a brief high-level comment, e.g.

> Unless a `step` value is explicitly passed, look for any unit-specific values. For instance, a sensible step value for `em` might be `0.1` instead of `1`.

Reply:
Thanks for the feedback, Ill look to update all this again tomorrow.

Reply:
updated, thanks